### ISSUE TITLE
0.20 zip release packaging & install documentation

### ISSUE THREAD

**Issue by Daryes** [node-red/node-red#2144]
Related to the getting-started/installation documentation, using the release file in the Alternative methods : 
https://nodered.org/docs/getting-started/installation

0.19.x releases were packaged like specified in the documentation. For example, [0.19.6](https://github.com/node-red/node-red/releases/download/0.19.6/node-red-0.19.6.zip) has a single directory at the first level in the zip archive, and everything required by node-red is under the said directory.  

0.20 releases use a different structure, the zip archive currently contains 2 directories :
- node-red
- @node-red

For what I saw, there's no information in the documentation about the @node-red directory.  
After installing only the node-red directory and checking how nodejs/npm managed the modules, it seems the @node-red directory in the zip archive is not at the right place and/or completely skipped due to the `node-red/package.json` .

**Comment by knolleary** [comment]
Yes the zip layout has changed and the docs need to update to reflect this fact.

However, as we have now split node-red into multiple modules (which has been mentioned at length in the release blog post and other posts) it makes much less sense to download the zip when running `npm install node-red` will get you everything you need properly installed.

Did you have a particular dependency on the zip file?

Also, *please* use the issue template in the future.

**Comment by Daryes** [comment]
I don't use `npm install <module>`, nor a global install. I'm having npm working directly on the zip content as a source.  
This allow me to control the installed version and the location : node-red is self-contained with all the dependencies under a `release/vX.XX.XX` directory with a symlink targeting the lastest version.

A little grease is required for the first installation, but this allows, amongst other things, to roll back to another version with just a change to the symlink (assuming there weren't any modification to the data structure).
 
The only problem was to know what to do with the @node-red directory, and if it was required or not.   Given the package.json file has enough information, npm is happy without this directory, or if moved under node-red/node_modules/


Sorry for the template, given it was specific to the new packaging, I didn't think it was required.
I've added at least the version information.

**Comment by knolleary** [comment]
Closing as part of a tidy-up of the issue tracker — this has been inactive for a long time and much has changed in Node-RED since. If it's still relevant on a current version, please open a fresh issue with up-to-date details and we'll take a look. Thanks for raising it.
